figures/metrics.py

```diff
@@ -436,6 +436,28 @@ def calc_from_user_model():
     return calc_from_user_model()
 
 
+def get_total_site_staff_users_for_time_period(site, start_date, end_date):
+    """
+    Returns the total number of staff users for the time period
+    """
+
+    def calc_from_user_model():
+        """
+        Calculate the total number of staff users for the time period
+        """
+        filter_args = dict(
+            date_joined__date__lt=next_day(end_date),
+        )
+        users = figures.sites.get_users_for_site(site).filter(
+            courseaccessrole__role='global_course_creator',
+            is_superuser=False
```

Comment:
need to add `is_staff=False` to be on safe side.

Reply:
Done